project/profiles/capacity4more/modules/c4m/field/c4m_field_og/includes/c4m_field_og.theme.node.inc

```diff
@@ -23,7 +23,7 @@ function theme_c4m_field_og_group($field) {
 
   // Don't show this field in a group context.
   $context = og_context();
-  if ($context) {
+  if ($context && $field['entity']->type !== "share") {
```

Comment:
@soniCaH 
after new installation on develop, going to __Nobel Prize__ group, I get this errors:

![selection_307](https://cloud.githubusercontent.com/assets/5985683/19684477/087e9500-9ac0-11e6-90b5-92554c9ebea4.png)